sklearn/metrics/cluster/tests/test_unsupervised.py

```diff
@@ -22,7 +22,6 @@ def test_silhouette():
     X_lil = sp.lil_matrix(X_dense)
     y = dataset.target
 
-    X_dense_score = {}
     for X in [X_dense, X_csr, X_dok, X_lil]:
```

Comment:
That's not how `accept_sparse` works: "If the input is sparse but not in the allowed format, it will be converted to the first listed format."